app/policies/security_question_policy.rb

```diff
@@ -0,0 +1,16 @@
+class SecurityQuestionPolicy < ApplicationPolicy
+
+  def edit?
+    has_security_questions?
+  end
+
+  def update?
+    has_security_questions?
+  end
+
+  private
+
+  def has_security_questions?
+    user && !user.portal_student.nil?
+  end
```

Comment:
I'd change this method name to `is_student?`. Might even move it to the application_policy, but I suspect it won't be used anywhere else.